client/jetpack-cloud/sections/settings/advanced-credentials/verification/index.tsx

```diff
@@ -27,22 +44,50 @@ const Verification: FunctionComponent = () => {
 		() => {
 			setCurrentStep( currentStep + 1 );
 		},
-		currentStep < steps.length - 1 ? random( EVERY_SECOND, EVERY_SECOND * 3 ) : null
+		currentStep < steps.length ? random( EVERY_SECOND, EVERY_SECOND * 3 ) : null
 	);
 
 	return (
 		<div>
-			<h4>Verification</h4>
-			<ul>
+			<div className="verification__title">
+				<h3>
+					{ translate( 'Establishing a connection to {{strong}}%(siteSlug)s{{/strong}}.', {
+						args: {
+							siteSlug,
+						},
+						components: {
+							strong: <strong />,
+						},
+					} ) }
+				</h3>
+			</div>
+			<ul className="verification__step-list">
 				{ steps.map( ( step, index ) => {
 					if ( index < currentStep ) {
-						return <li key={ index }>{ step }</li>;
+						return (
+							<li key={ index } className="verification__step-complete">
+								<Gridicon icon="checkmark" />
+								{ step }
+							</li>
+						);
 					} else if ( index === currentStep ) {
-						return <li key={ index }>{ step }</li>;
+						return (
+							<li key={ index } className="verification__step-in-progress">
+								<Gridicon icon="sync" />
+								{ step }
+							</li>
+						);
 					}
 					return null;
 				} ) }
 			</ul>
+			{ currentStep >= steps.length && (
+				<div className="verification__buttons">
+					<Button primary onClick={ onFinishUp }>
+						{ translate( 'Finish up' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Finishing up!', { context: 'Shown during a jetpack authorization process, while we retrieve the info we need to show the last page'} )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-cloud-update-new-credentials-styles)